In [66]:
import pandas as pd

In [67]:
df7 = pd.read_csv('df7_code.csv')
df7

Unnamed: 0        변수명              변수설명
0              0      mod_d         최종 DB 수정일
1              1         ID            개인 아이디
2              2     ID_fam            가구 아이디
3              3       year              조사연도
4              4     region            17개 시도
...          ...        ...               ...
1441         142     HEI_NA            나트륨 섭취
1442         143  HEI_SWEET  당류, 음료류의 에너지섭취비율
1443         144    HEI_CHO      탄수화물 에너지섭취비율
1444         145    HEI_FAT        지방 에너지섭취비율
1445         146     HEI_EN            에너지 섭취

[1446 rows x 3 columns]

In [68]:
df8 = pd.read_csv('df8_code.csv')
df8

Unnamed: 0         변수명                                변수설명
0             0       mod_d                           최종 DB 수정일
1             1          ID                              개인 아이디
2             2      ID_fam                              가구 아이디
3             3        year                                조사연도
4             4      region                              17개 시도
..          ...         ...                                 ...
900         130    LF_S14_1         식비가 부족하여 가구 내 아동이 식사를 거른 빈도
901         131      LF_S15  식비가 부족하여 가구 내 아동이 배가 고픈데도 먹지 못한 경험
902         132      LF_S16       식비가 부족하여 가구 내 아동이 하루 종일 굶은 경험
903         133    LF_SECUR                            식품안정성 점수
904         134  LF_SECUR_G                        수준별 식품안정성 정도

[905 rows x 3 columns]

In [69]:
df7.columns

Index(['Unnamed: 0', '변수명', '변수설명'], dtype='object')

In [70]:
df7 = df7.drop(['Unnamed: 0'], axis=1)
df8 = df8.drop(['Unnamed: 0'], axis=1)

In [71]:
df7

변수명              변수설명
0         mod_d         최종 DB 수정일
1            ID            개인 아이디
2        ID_fam            가구 아이디
3          year              조사연도
4        region            17개 시도
...         ...               ...
1441     HEI_NA            나트륨 섭취
1442  HEI_SWEET  당류, 음료류의 에너지섭취비율
1443    HEI_CHO      탄수화물 에너지섭취비율
1444    HEI_FAT        지방 에너지섭취비율
1445     HEI_EN            에너지 섭취

[1446 rows x 2 columns]

In [72]:
df8

변수명                                변수설명
0         mod_d                           최종 DB 수정일
1            ID                              개인 아이디
2        ID_fam                              가구 아이디
3          year                                조사연도
4        region                              17개 시도
..          ...                                 ...
900    LF_S14_1         식비가 부족하여 가구 내 아동이 식사를 거른 빈도
901      LF_S15  식비가 부족하여 가구 내 아동이 배가 고픈데도 먹지 못한 경험
902      LF_S16       식비가 부족하여 가구 내 아동이 하루 종일 굶은 경험
903    LF_SECUR                            식품안정성 점수
904  LF_SECUR_G                        수준별 식품안정성 정도

[905 rows x 2 columns]

In [73]:
col_7 = list(df7['변수명'])
col_8 = list(df8['변수명'])

In [74]:
# 7차, 8차 변수정의서에서 '변수명'들을 대문자화
col_7_up = []
col_8_up = []

for i in col_7:
    col_7_up.append(i.upper().strip())

for i in col_8:
    col_8_up.append(i.upper().strip())

In [75]:
col_7_up

['MOD_D',
 'ID',
 'ID_FAM',
 'YEAR',
 'REGION',
 'TOWN_T',
 'APT_T',
 'PSU',
 'SEX',
 'AGE',
 'AGE_MONTH',
 'INCM',
 'HO_INCM',
 'INCM5',
 'HO_INCM5',
 'EDU',
 'OCCP',
 'WT_HS',
 'WT_ITVEX',
 'WT_PFT',
 'WT_HM',
 'WT_VT',
 'WT_NN',
 'WT_EX1',
 'WT_NTR',
 'WT_TOT',
 'WT_PFHM',
 'WT_PFVT',
 'WT_PFNT',
 'WT_HMNT',
 'WT_VTNT',
 'WT_NNNT',
 'WT_PFHMNT',
 'WT_PFVTNT',
 'WT_EX1NT',
 'WT_EX1HM',
 'WT_EX1PF',
 'WT_TOT1',
 'WT_EX1PFNT',
 'WT_EX1HMNT',
 'WT_TOT1NT',
 'WT_OE',
 'WT_OENT',
 'KSTRATA',
 'CFAM',
 'GENERTN',
 'ALLOWNC',
 'HOUSE',
 'LIVE_T',
 'AINC_UNIT1',
 'AINC_1',
 'AINC',
 'MARRI_1',
 'MARRI_2',
 'FAM_RELA',
 'TINS',
 'NPINS',
 'ID_F',
 'ID_M',
 'D_1_1',
 'D_2_1',
 'D_2_WK',
 'DI1_DG',
 'DI1_AG',
 'DI1_PR',
 'DI1_PT',
 'DI1_2',
 'DI2_DG',
 'DI2_AG',
 'DI2_PR',
 'DI2_PT',
 'DI2_2',
 'DI3_DG',
 'DI3_AG',
 'DI3_PR',
 'DI3_PT',
 'DI3_2',
 'DI4_DG',
 'DI4_PR',
 'DI4_PT',
 'DI5_DG',
 'DI5_AG',
 'DI5_PR',
 'DI5_PT',
 'DI6_DG',
 'DI6_AG',
 'DI6_PR',
 'DI6_PT',
 'DM1_DG',
 'DM1_PR',
 'DM1_P

In [76]:
col_8_up

['MOD_D',
 'ID',
 'ID_FAM',
 'YEAR',
 'REGION',
 'TOWN_T',
 'APT_T',
 'PSU',
 'SEX',
 'AGE',
 'AGE_MONTH',
 'INCM',
 'HO_INCM',
 'INCM5',
 'HO_INCM5',
 'EDU',
 'OCCP',
 'WT_HS',
 'WT_ITVEX',
 'WT_OE',
 'WT_PFT',
 'WT_EX1',
 'WT_IGE',
 'WT_NTR',
 'WT_TOT',
 'WT_PFNT',
 'WT_EX1NT',
 'WT_IGENT',
 'KSTRATA',
 'CFAM',
 'GENERTN',
 'ALLOWNC',
 'HOUSE',
 'LIVE_T',
 'AINC_UNIT1',
 'AINC_1',
 'AINC',
 'MARRI_1',
 'MARRI_2',
 'FAM_RELA',
 'TINS',
 'NPINS',
 'ID_F',
 'ID_M',
 'D_1_1',
 'D_2_1',
 'D_2_WK',
 'DI1_DG',
 'DI1_AG',
 'DI1_PR',
 'DI1_PT',
 'DI1_2',
 'DI2_DG',
 'DI2_AG',
 'DI2_PR',
 'DI2_PT',
 'DI2_2',
 'DI3_DG',
 'DI3_AG',
 'DI3_PR',
 'DI3_PT',
 'DI3_2',
 'DI4_DG',
 'DI4_PR',
 'DI4_PT',
 'DI5_DG',
 'DI5_AG',
 'DI5_PR',
 'DI5_PT',
 'DI6_DG',
 'DI6_AG',
 'DI6_PR',
 'DI6_PT',
 'DM1_DG',
 'DM1_PR',
 'DM1_PT',
 'DM2_DG',
 'DM2_AG',
 'DM2_PR',
 'DM2_PT',
 'DM3_DG',
 'DM3_AG',
 'DM3_PR',
 'DM3_PT',
 'DM4_DG',
 'DM4_AG',
 'DM4_PR',
 'DM4_PT',
 'DJ2_DG',
 'DJ2_AG',
 'DJ2_PR',
 'DJ2_PT',
 'DJ4_DG

In [77]:
df7['변수명'] = col_7_up
df8['변수명'] = col_8_up

In [78]:
df7

변수명              변수설명
0         MOD_D         최종 DB 수정일
1            ID            개인 아이디
2        ID_FAM            가구 아이디
3          YEAR              조사연도
4        REGION            17개 시도
...         ...               ...
1441     HEI_NA            나트륨 섭취
1442  HEI_SWEET  당류, 음료류의 에너지섭취비율
1443    HEI_CHO      탄수화물 에너지섭취비율
1444    HEI_FAT        지방 에너지섭취비율
1445     HEI_EN            에너지 섭취

[1446 rows x 2 columns]

In [79]:
df8

변수명                                변수설명
0         MOD_D                           최종 DB 수정일
1            ID                              개인 아이디
2        ID_FAM                              가구 아이디
3          YEAR                                조사연도
4        REGION                              17개 시도
..          ...                                 ...
900    LF_S14_1         식비가 부족하여 가구 내 아동이 식사를 거른 빈도
901      LF_S15  식비가 부족하여 가구 내 아동이 배가 고픈데도 먹지 못한 경험
902      LF_S16       식비가 부족하여 가구 내 아동이 하루 종일 굶은 경험
903    LF_SECUR                            식품안정성 점수
904  LF_SECUR_G                        수준별 식품안정성 정도

[905 rows x 2 columns]

In [80]:
data_16_1 = pd.read_csv('C:/Users/User/VS_code/sas_view/data/pre2_age16_1.csv')
data_16_2 = pd.read_csv('C:/Users/User/VS_code/sas_view/data/pre2_age16_2.csv')
data_17_1 = pd.read_csv('C:/Users/User/VS_code/sas_view/data/pre2_age17_1.csv')
data_17_2 = pd.read_csv('C:/Users/User/VS_code/sas_view/data/pre2_age17_2.csv')
data_18_1 = pd.read_csv('C:/Users/User/VS_code/sas_view/data/pre2_age18_1.csv')
data_18_2 = pd.read_csv('C:/Users/User/VS_code/sas_view/data/pre2_age18_2.csv')
data_19_1 = pd.read_csv('C:/Users/User/VS_code/sas_view/data/pre2_age19_1.csv')
data_19_2 = pd.read_csv('C:/Users/User/VS_code/sas_view/data/pre2_age19_2.csv')
data_20_1 = pd.read_csv('C:/Users/User/VS_code/sas_view/data/pre2_age20_1.csv')
data_20_2 = pd.read_csv('C:/Users/User/VS_code/sas_view/data/pre2_age20_2.csv')

C:\Users\User\anaconda3\lib\site-packages\IPython\core\interactiveshell.py:3444: DtypeWarning: Columns (44,175,384,424,567) have mixed types.Specify dtype option on import or set low_memory=False.
  exec(code_obj, self.user_global_ns, self.user_ns)
C:\Users\User\anaconda3\lib\site-packages\IPython\core\interactiveshell.py:3444: DtypeWarning: Columns (251,297,413) have mixed types.Specify dtype option on import or set low_memory=False.
  exec(code_obj, self.user_global_ns, self.user_ns)
C:\Users\User\anaconda3\lib\site-packages\IPython\core\interactiveshell.py:3444: DtypeWarning: Columns (40,121,260,279,295,372,487,502) have mixed types.Specify dtype option on import or set low_memory=False.
  exec(code_obj, self.user_global_ns, self.user_ns)
C:\Users\User\anaconda3\lib\site-packages\IPython\core\interactiveshell.py:3444: DtypeWarning: Columns (40,41,95,598) have mixed types.Specify dtype option on import or set low_memory=False.
  exec(code_obj, self.user_global_ns, self.user_ns)
C:\Us

In [81]:
# 실제 데이터 16년~20년의 컬럼들을 리스트에 담아서 대문자화
data_16_1_up = []
data_16_2_up = []
data_17_1_up = []
data_17_2_up = []
data_18_1_up = []
data_18_2_up = []
data_19_1_up = []
data_19_2_up = []
data_20_1_up = []
data_20_2_up = []

for i in data_16_1.columns:
    data_16_1_up.append(i.upper())

for i in data_16_2.columns:
    data_16_2_up.append(i.upper())

for i in data_17_1.columns:
    data_17_1_up.append(i.upper())

for i in data_17_2.columns:
    data_17_2_up.append(i.upper())

for i in data_18_1.columns:
    data_18_1_up.append(i.upper())

for i in data_18_2.columns:
    data_18_2_up.append(i.upper())

for i in data_19_1.columns:
    data_19_1_up.append(i.upper())

for i in data_19_2.columns:
    data_19_2_up.append(i.upper())

for i in data_20_1.columns:
    data_20_1_up.append(i.upper())

for i in data_20_2.columns:
    data_20_2_up.append(i.upper())

In [82]:
data_up = data_16_1_up + data_16_2_up +data_17_1_up + data_17_2_up + data_18_1_up + data_18_2_up +data_19_1_up + data_19_2_up +data_20_1_up + data_20_2_up 
data_up
len(data_up)

3562

In [83]:
data_up

['ID',
 'ID_FAM',
 'REGION',
 'TOWN_T',
 'APT_T',
 'PSU',
 'SEX',
 'AGE',
 'INCM',
 'HO_INCM',
 'INCM5',
 'HO_INCM5',
 'EDU',
 'OCCP',
 'WT_HS',
 'WT_ITVEX',
 'WT_PFT',
 'WT_HM',
 'WT_NTR',
 'WT_TOT',
 'WT_PFHM',
 'WT_PFNT',
 'WT_HMNT',
 'WT_PFHMNT',
 'WT_NN',
 'WT_NNNT',
 'WT_VT',
 'WT_PFVT',
 'WT_VTNT',
 'WT_PFVTNT',
 'KSTRATA',
 'CFAM',
 'GENERTN',
 'ALLOWNC',
 'HOUSE',
 'LIVE_T',
 'AINC_UNIT1',
 'AINC_1',
 'AINC',
 'MARRI_1',
 'MARRI_2',
 'FAM_RELA',
 'TINS',
 'NPINS',
 'ID_M',
 'D_1_1',
 'D_2_1',
 'D_2_WK',
 'DI1_DG',
 'DI1_AG',
 'DI1_PT',
 'DI1_2',
 'DI2_DG',
 'DI2_AG',
 'DI2_PT',
 'DI2_2',
 'DI3_DG',
 'DI3_AG',
 'DI3_PT',
 'DI3_2',
 'DI4_DG',
 'DI4_PT',
 'DI5_DG',
 'DI5_AG',
 'DI5_PT',
 'DI6_DG',
 'DI6_AG',
 'DI6_PT',
 'DM1_DG',
 'DM1_PR',
 'DM1_PT',
 'DM2_DG',
 'DM2_AG',
 'DM2_PT',
 'DM3_DG',
 'DM3_AG',
 'DM3_PT',
 'DM4_DG',
 'DM4_AG',
 'DM4_PT',
 'DJ2_DG',
 'DJ2_AG',
 'DJ2_PT',
 'DJ4_DG',
 'DJ4_AG',
 'DJ4_PT',
 'DJ4_3',
 'DE1_DG',
 'DE1_AG',
 'DE1_PT',
 'DE1_3',
 'DE1_31',
 'D

In [84]:
data_up = list(set(data_up))
len(data_up)

916

In [85]:
col_7_up

['MOD_D',
 'ID',
 'ID_FAM',
 'YEAR',
 'REGION',
 'TOWN_T',
 'APT_T',
 'PSU',
 'SEX',
 'AGE',
 'AGE_MONTH',
 'INCM',
 'HO_INCM',
 'INCM5',
 'HO_INCM5',
 'EDU',
 'OCCP',
 'WT_HS',
 'WT_ITVEX',
 'WT_PFT',
 'WT_HM',
 'WT_VT',
 'WT_NN',
 'WT_EX1',
 'WT_NTR',
 'WT_TOT',
 'WT_PFHM',
 'WT_PFVT',
 'WT_PFNT',
 'WT_HMNT',
 'WT_VTNT',
 'WT_NNNT',
 'WT_PFHMNT',
 'WT_PFVTNT',
 'WT_EX1NT',
 'WT_EX1HM',
 'WT_EX1PF',
 'WT_TOT1',
 'WT_EX1PFNT',
 'WT_EX1HMNT',
 'WT_TOT1NT',
 'WT_OE',
 'WT_OENT',
 'KSTRATA',
 'CFAM',
 'GENERTN',
 'ALLOWNC',
 'HOUSE',
 'LIVE_T',
 'AINC_UNIT1',
 'AINC_1',
 'AINC',
 'MARRI_1',
 'MARRI_2',
 'FAM_RELA',
 'TINS',
 'NPINS',
 'ID_F',
 'ID_M',
 'D_1_1',
 'D_2_1',
 'D_2_WK',
 'DI1_DG',
 'DI1_AG',
 'DI1_PR',
 'DI1_PT',
 'DI1_2',
 'DI2_DG',
 'DI2_AG',
 'DI2_PR',
 'DI2_PT',
 'DI2_2',
 'DI3_DG',
 'DI3_AG',
 'DI3_PR',
 'DI3_PT',
 'DI3_2',
 'DI4_DG',
 'DI4_PR',
 'DI4_PT',
 'DI5_DG',
 'DI5_AG',
 'DI5_PR',
 'DI5_PT',
 'DI6_DG',
 'DI6_AG',
 'DI6_PR',
 'DI6_PT',
 'DM1_DG',
 'DM1_PR',
 'DM1_P

In [86]:
col_8_up

['MOD_D',
 'ID',
 'ID_FAM',
 'YEAR',
 'REGION',
 'TOWN_T',
 'APT_T',
 'PSU',
 'SEX',
 'AGE',
 'AGE_MONTH',
 'INCM',
 'HO_INCM',
 'INCM5',
 'HO_INCM5',
 'EDU',
 'OCCP',
 'WT_HS',
 'WT_ITVEX',
 'WT_OE',
 'WT_PFT',
 'WT_EX1',
 'WT_IGE',
 'WT_NTR',
 'WT_TOT',
 'WT_PFNT',
 'WT_EX1NT',
 'WT_IGENT',
 'KSTRATA',
 'CFAM',
 'GENERTN',
 'ALLOWNC',
 'HOUSE',
 'LIVE_T',
 'AINC_UNIT1',
 'AINC_1',
 'AINC',
 'MARRI_1',
 'MARRI_2',
 'FAM_RELA',
 'TINS',
 'NPINS',
 'ID_F',
 'ID_M',
 'D_1_1',
 'D_2_1',
 'D_2_WK',
 'DI1_DG',
 'DI1_AG',
 'DI1_PR',
 'DI1_PT',
 'DI1_2',
 'DI2_DG',
 'DI2_AG',
 'DI2_PR',
 'DI2_PT',
 'DI2_2',
 'DI3_DG',
 'DI3_AG',
 'DI3_PR',
 'DI3_PT',
 'DI3_2',
 'DI4_DG',
 'DI4_PR',
 'DI4_PT',
 'DI5_DG',
 'DI5_AG',
 'DI5_PR',
 'DI5_PT',
 'DI6_DG',
 'DI6_AG',
 'DI6_PR',
 'DI6_PT',
 'DM1_DG',
 'DM1_PR',
 'DM1_PT',
 'DM2_DG',
 'DM2_AG',
 'DM2_PR',
 'DM2_PT',
 'DM3_DG',
 'DM3_AG',
 'DM3_PR',
 'DM3_PT',
 'DM4_DG',
 'DM4_AG',
 'DM4_PR',
 'DM4_PT',
 'DJ2_DG',
 'DJ2_AG',
 'DJ2_PR',
 'DJ2_PT',
 'DJ4_DG

In [87]:
col_up = col_7_up + col_8_up
col_up = list(set(col_up))
col_up

['NF_TDF',
 'FA_J_BUDA',
 'DM8_PR',
 'BS12_42',
 'FS_BEER',
 'AC8_3W_03',
 'WT_PFT',
 'MARRI_1',
 'FQ_PEAR',
 'HE_SBP3',
 'NF_RETIN',
 'DF1_YA',
 'AC3_3_05',
 'T_HR_EX_R_2',
 'DI5_PT',
 'FQ_MILK',
 'T_Q_SNST4',
 'DI4_PR',
 'HE_VITE',
 'MOD_D',
 'NF_22_1',
 'NS_CA',
 'DC3_PR',
 'FF_CITRUS_YR',
 'FQ_HAIRT',
 'FF_BELLFI',
 'LF_S14_1',
 'DF2_DG',
 'HE_LDL_DRCT',
 'FA_PEANUT',
 'FQ_MUSHRO',
 'NF_12_0',
 'HE_ANEM',
 'Y_MTM_S2',
 'FA_R_BEEF',
 'Y_MTM_D1',
 'BE3_81',
 'DH4_AG',
 'NF_20_1',
 'BM1_2',
 'FF_BUNO',
 'APT_T',
 'DI6_AG',
 'DE2_DG',
 'FF_MMELON',
 'BP16_21',
 'O_BR_N_L',
 'NO_CPI_34',
 'FQ_FJUICE',
 'FA_T_POTA',
 'O_DMFIP',
 'FF_BUTTER',
 'DM2_PR',
 'LQ4_10',
 'GS_MEA_R_3',
 'AC8_2W_04',
 'DI3_AG',
 'BD2_1',
 'DH2_DG',
 'T_IM_A_R',
 'AC3_2_04',
 'WT_ITVEX',
 'DI6_PR',
 'T_HR_EX_L_1',
 'FQ_SOJU',
 'EQ5D',
 'NF_16_1',
 'FQ_WMELON',
 'DC6_PR',
 'FQ_S_CHIC',
 'OR1_2',
 'DE1_DG',
 'HE_COUGH2',
 'LQ_6HT',
 'FF_RCAK',
 'DA_CAFFEINE',
 'LW_MT',
 'EDUC',
 'N_TD_WT',
 'DM8_DG',
 'FS_CITRUS',
 

In [88]:
len(col_up)

1605

In [89]:
col_up[0]

'NF_TDF'

In [94]:
col_not_in_total = []
col_in_total = []

for i in data_up:
    if i in col_up:
        col_in_total.append(i)
    else:
        col_not_in_total.append(i)

print(col_not_in_total)
print(col_in_total)

print(len(col_not_in_total))
print(len(col_in_total))

['T_HL_MODERATE', 'T_BALANCE']
['BO3_05', 'E_CR_M', 'T_Q_CR3', 'DI2_PT', 'T_NQ_FIR', 'INCM5', 'DE1_PR', 'LQ_1HT', 'BP16_11', 'DI5_DG', 'AINC_1', 'BS12_42', 'MO4_11', 'DM8_PR', 'LQ1_MN', 'BD7_67', 'HE_NC', 'T_Q_DZ2', 'N_NA', 'LQ4_00', 'E_CL_M', 'WT_PFT', 'T_HR_EX_R_1', 'MARRI_1', 'MO1_WK', 'BA2_14', 'T_TYMP_DIS', 'LQ4_01', 'BS12_47_4', 'AC3_2_01', 'HE_SBP3', 'HE_DMDG', 'T_NQ_OCP_T', 'LF_S9_1', 'T_EX', 'DN6_YA', 'BE3_72', 'BD2_31', 'DC4_AG', 'SEX', 'ALLOWNC', 'BM2_3', 'BO3_10', 'N_FOLATE', 'T_HR500_PASS', 'DF1_YA', 'LQ_2EQL', 'BO1_2', 'HE_PEF', 'T_HR_EX_R_2', 'DJ8_PR', 'DI2_2', 'DI5_PT', 'N_SFA', 'T_HR1000_RT', 'HE_HEPAB', 'T_Q_SNST4', 'HE_COPD', 'BM13_5', 'DM8_PT', 'AC8_2_01', 'DK4_AG', 'T_Q_DZ4', 'T_HR2000_LT', 'DI4_PR', 'BS3_2', 'DF2_PT', 'HE_VITE', 'BE3_83', 'DC3_PR', 'BS5_4', 'WT_PFHM', 'T_IM_B_L', 'LF_S14_1', 'BS10_2', 'DJ6_PT', 'BE8_2', 'BS12_44', 'HE_BUN', 'MO4_12', 'GS_USE', 'DH3_PR', 'E_PEO', 'DF2_DG', 'BS12_35', 'EC_WHT_23', 'HE_LDL_DRCT', 'BS5', 'BO3_02', 'HE_IHDFH1', 'DK8_PT

#### 실제 데이터에 있는 컬럼명(16년~20년 변수들 합쳐서 중복 제거한 것) = 916
#### 실제 데이터에는 있는데 변수설명서에는 없는 컬럼명 = 2
#### 실제 데이터에도 있고 변수설명서에도 있는 컬럼명 = 914

In [93]:
print(len(col_up))
print(len(data_up))

1605
916


In [92]:
total_not_in_col = []
total_in_col = []

for i in col_up:
    if i in data_up:
        total_in_col.append(i)
    else:
        total_not_in_col.append(i)

print(total_not_in_col)
print(total_in_col)

print(len(total_not_in_col))
print(len(total_in_col))

['NF_TDF', 'FA_J_BUDA', 'FS_BEER', 'AC8_3W_03', 'FQ_PEAR', 'NF_RETIN', 'AC3_3_05', 'FQ_MILK', 'MOD_D', 'NF_22_1', 'NS_CA', 'FF_CITRUS_YR', 'FQ_HAIRT', 'FF_BELLFI', 'FA_PEANUT', 'FQ_MUSHRO', 'NF_12_0', 'Y_MTM_S2', 'FA_R_BEEF', 'Y_MTM_D1', 'NF_20_1', 'FF_BUNO', 'FF_MMELON', 'O_BR_N_L', 'NO_CPI_34', 'FQ_FJUICE', 'FA_T_POTA', 'FF_BUTTER', 'AC8_2W_04', 'AC3_2_04', 'FQ_SOJU', 'NF_16_1', 'FQ_WMELON', 'FQ_S_CHIC', 'FF_RCAK', 'DA_CAFFEINE', 'N_TD_WT', 'FS_CITRUS', 'FA_PEAR', 'FF_MMELON_YR', 'FQ_BNCURD', 'DQ_YOGURT', 'FA_S_RCAK', 'AC8_1E_09', 'FF_J_SOYP', 'FQ_BARLEY', 'FF_PEAR', 'AC8_2_04', 'FA_PORKBY', 'FQ_KIMCHI', 'FQ_FAT', 'FF_S_BEAN', 'FQ_BANANA', 'NS_WT', 'FF_S_SWED', 'FF_F_PORK', 'AC8_3_07', 'FF_CAKE', 'FQ_T_POTA', 'N_PRODUCT', 'FQ_R_PORK', 'NF_22_5', 'NF_CA', 'FA_MACKER', 'FF_WHNO', 'FQ_L_YOGU', 'FS_COFFEE', 'FF_TANG', 'FA_SCRAB', 'FA_S_SWED', 'FQ_JAM', 'FQ_NA', 'FF_F_EGG', 'FF_J_POLL', 'FA_S_CHIC', 'AC8_1_09', 'NF_18_4', 'FA_J_SOYP', 'FF_BANANA', 'FQ_VITC', 'DQ_FVJUICE', 'AC3_3E_07', 'FQ

#### 변수설명서에 있는 컬럼명(7차, 8차 변수설명서 합치고 중복 제거한 것) = 1605
#### 변수설명서에는 있는데 실제 데이터에는 없는 컬럼명 = 691
#### 변수설명서에도 있고 실제 데이터에는 있는 컬럼명 = 914

## <실제 데이터에 있는 컬럼명(16년~20년 변수들 합쳐서 중복 제거한 것)> = 916
### 실제 데이터에는 있는데 변수설명서에는 없는 컬럼명 = 2
### 실제 데이터에도 있고 변수설명서에도 있는 컬럼명 = 914

## <변수설명서에 있는 컬럼명(7차, 8차 변수설명서 합치고 중복 제거한 것)> = 1605
### 변수설명서에는 있는데 실제 데이터에는 없는 컬럼명 = 691
### 변수설명서에도 있고 실제 데이터에는 있는 컬럼명 = 914